src/fileCLI/fileCLI.hpp

```diff
@@ -93,9 +93,21 @@ class FileCLI{
     DIR* dir_stack[FILE_CLI_MAX_DIR_DEPTH];
     char path_stack[FILE_CLI_MAX_DIR_DEPTH][NAME_MAX];
     int current_dir;
+
+    /**
+     * @brief Structure representing a command menu entry for FileCLI.
+     *
+     */
     typedef struct menu_
     {
+        /**
+         * A character representing a user command. Will trigger a specific function.
+         */
         const char cmd;
+        /**
+         * A pointer to a place in ram where a function with properties is held. Executes the
+         * command associated with cmd.
+         */
```

Comment:
Formally, this is the execution function for this command.  It does not necessarily need to be in RAM - this could be in flash memory, and realistically probably is in flash.

Read through the block of code at https://github.com/UCSD-E4E/smartfin-fw3/blob/0da8c2cc2e3a33847b503ccdb7f550c4fd84b42e/src/fileCLI/fileCLI.cpp#L66 to understand how this is being used.